core_backend/app/schemas.py

```diff
@@ -103,42 +109,78 @@ class FeedbackBase(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
 
-class ContentCreate(BaseModel):
+class LanguageBase(BaseModel):
+    """
+    Pydantic model for language
+    """
+
+    language_name: str
+    is_default: bool
+    model_config = ConfigDict(from_attributes=True)
+
+    @validator("language_name")
+    def language_name_must_be_valid(cls, value: str) -> str:
+        # Regex pattern allows letters, spaces, hyphens, and apostrophes
+        if not re.match(r"^[A-Za-zÀ-ÖØ-öø-ÿ\-' ]+$", value):
+            raise ValueError("Invalid characters in language name.")
+        return value
+
+
+class LanguageRetrieve(LanguageBase):
+    """
+    Pydantic model for language retrieval
+    """
+
+    language_id: int
+    created_datetime_utc: datetime
+    updated_datetime_utc: datetime
+
+    model_config = ConfigDict(from_attributes=True)
+
+
+class ContentTextCreate(BaseModel):
     """
     Pydantic model for content creation
     """
 
     # Ensure len("*{title}*\n\n{text}") <= 1600
     content_title: Annotated[str, StringConstraints(max_length=150)]
     content_text: Annotated[str, StringConstraints(max_length=2000)]
-    content_language: str = "ENGLISH"
+    language_id: int
+    content_id: Optional[int] = None
```

Comment:
Can we rename this to `existing_content_id`?
This way it is less confusing to users. Users may supply `content_id` even for a content that doesn't exist, and receive a new `content_id` like the following:
<img width="512" alt="image" src="https://github.com/IDinsight/aaq-core/assets/7042047/f402e41b-3c88-4fda-aa1c-5a4d670e8608">

Reply:
Also some description will be helpful!
```python
from pydantic import BaseModel, Field
```

```suggestion
    existing_content_id: Optional[int] = Field(default=None, description='If adding or editing content text to an existing content, provide its ID')
```